common/src/main/java/tachyon/conf/TachyonConf.java

```diff
@@ -134,6 +135,29 @@ public TachyonConf() {
       throw new RuntimeException("Unable to load default Tachyon properties file.", e);
     }
 
+    // Override default properties if there exist corresponding environment variables
+    if (getenv("TACHYON_MASTER_ADDRESS") != null) {
+      defaultProps.setProperty(Constants.MASTER_HOSTNAME, getenv("TACHYON_MASTER_ADDRESS"));
+    }
+    if (getenv("TACHYON_MASTER_BIND_HOST") != null) {
+      defaultProps.setProperty(Constants.MASTER_BIND_HOST, getenv("TACHYON_MASTER_BIND_HOST"));
+    }
+    if (getenv("TACHYON_MASTER_WEB_BIND_HOST") != null) {
+      defaultProps.setProperty(Constants.MASTER_WEB_BIND_HOST,
+          getenv("TACHYON_MASTER_WEB_BIND_HOST"));
+    }
+    if (getenv("TACHYON_WORKER_BIND_HOST") != null) {
+      defaultProps.setProperty(Constants.WORKER_BIND_HOST, getenv("TACHYON_WORKER_BIND_HOST"));
+    }
+    if (getenv("TACHYON_WORKER_DATA_BIND_HOST") != null) {
+      defaultProps.setProperty(Constants.WORKER_DATA_BIND_HOST,
+          getenv("TACHYON_WORKER_DATA_BIND_HOST"));
+    }
+    if (getenv("TACHYON_WORKER_WEB_BIND_HOST") != null) {
+      defaultProps.setProperty(Constants.WORKER_WEB_BIND_HOST,
+          getenv("TACHYON_WORKER_WEB_BIND_HOST"));
+    }
+
```

Comment:
Actually, we only need to select one of configuration properties for loading environment variables. I select defaultProps in current design. If we use systemProps, we have to hardcode all environment variables into "TACHYON_JAVA_OPTS+=".